### DISCUSSION TITLE
How do I create record with file field during migration?

### DISCUSSION THREAD

**Discussion by chungphing** [pocketbase/pocketbase#5376] [Q&A]
Hi, I'm trying to seed some data to my collections that contain images. But it seems like I cannot import "fs" module. 

I could upload file from pocketbase sdk but I'm at a lost on how to do it in the migration files.

```
const recipeImage = {
      image: imageFile // how should I create the file field here?
    };
const record = new Record(collection, recipeImage);
 dao.saveRecord(record)
```

**Comment by ganigeorgiev** [comment]
There is no easy way to do that at the moment because we don't expose the `$app` instance within the migration which is required to init the app local or s3 filesystem.

This is already fixed as part of the [refactoring](https://github.com/pocketbase/pocketbase/discussions/4355) and will be available with the next v0.23 release soon (_there are no ETAs yet_).

A workaround could be instead of a migration to [register a console command](https://pocketbase.io/docs/js-console-commands/) and then save the record with the file using the RecordUpsert form as shown in https://pocketbase.io/docs/js-records/#create-new-record-with-data-validations.
